gamestonk_terminal/config_terminal.py

```diff
@@ -15,6 +15,30 @@
 # By default the jupyter notebook will be run on port 8888
 PAPERMILL_NOTEBOOK_REPORT_PORT = "8888"
 
+# Logging settings
+
+# 0 - INFO
+# 1 - DEBUG for terminal, INFO for libraries
+# 2 - DEBUG for terminal, DEBUG for libraries
+
+LOGGING_VERBOSITY = 2
+
+if tmp_verbosity := os.getenv("GT_LOGGING_VERBOSITY"):
```

Comment:
Why not following our other convention here? Like
```
LOGGING_VERBOSITY = os.getenv("GT_LOGGING_VERBOSITY") or "2"
```

Reply:
I am converting to int. Conversion to int may blow up.